README.md

```diff
@@ -16,7 +16,7 @@ The documentation for Visual Basic and Visual C# are located in a separate repo
 To contribute to this documentation, please see the [Contributing guide](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/CONTRIBUTING.md).
 We welcome your contributions to help us improve the Visual Studio docs. All the articles in this repository use GitHub flavored markdown.
 
-Several feature areas of Visual Studio have their own folders in this repo, such as **debugger** for topics on debugging, **ide** for topics on the Visual Studio interactive development environment (IDE), and so forth. The **/media** subfolder in each folder contains art files for the topics. The [Contributing guide](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/CONTRIBUTING.md) has more information.
+Several featured areas of Visual Studio have their own folders in this repo, such as **debugger** for topics on debugging, **ide** for topics on the Visual Studio interactive development environment (IDE), and so forth. The **/media** subfolder in each folder contains art files for the project. The [Contributing guide](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/CONTRIBUTING.md) has more information.
```

Comment:
@Vaibhav11Sajnani Thanks for contributing to the docs. I must respectfully decline these two word changes as the original words are actually the correct ones, at least for the terminology we use in our team.